app/gateway/changelog.md

```diff
@@ -3,6 +3,43 @@ title: Kong Gateway Changelog
 no_version: true
 ---
 
+<!-- vale off -->
+
+## 2.8.1.2
+**Release Date** 2022/06/28
+
+### Fixes
+
+#### Enterprise
+
+* Fixed an issue in hybrid mode where, if a service was set to `enabled: false` and that service had a route with an enabled plugin, any new data planes would receive empty configuration.
```

Comment:
```suggestion
* Fixed an issue in hybrid mode where, if a service was set to `enabled: false` and that service had a route with an enabled plugin, any new data planes would receive an empty configuration.
```

Reply:
Not sure I agree, since data planes is plural. I don't think that "a configuration" is correct either, so I'm going to leave this one as-is. "Data planes receive configuration from control planes" is a common statement for how Kong Gateway works.